README.rst

```diff
@@ -32,11 +32,15 @@ Building the code
 
 Follow the usual CMake build workflow:
 
-* Configure the project, specify your compiler (e.g. ``gfortran``), the install
+* Configure the project, specify your compiler (e.g. ``gfortran``, ``ifort``, etc), the install
   location (e.g. ``$HOME/opt/skprogs``) and the build directory
   (e.g. ``_build``)::
 
-    FC=gfortran cmake -DCMAKE_INSTALL_PREFIX=$HOME/opt/skprogs -B _build .
+    FC=gfortran cmake -DCMAKE_INSTALL_PREFIX=$HOME/opt/skprogs -DCMAKE_Fortran_FLAGS=-fopenmp -B _build .
```

Comment:
Please remove the `-fopenmp` flag. OpenMP support should be added via OpenMP-detection in CMake and linking against the `OpenMP::OpenMP_Fortran` target.